devices/sonoff.rst

```diff
@@ -310,6 +310,20 @@ Shelly 2
     GPIO12, SW Input #1,
     GPIO14, SW Input #2,
 
+Shelly 2.5
+--------
```

Comment:
```suggestion
----------
```

Title underline must match title length

Reply:
Didn't know that, I have a lot to learn about Github.
Thanks, is there anything else I should do for this?

Reply:
@Taigar2015 LGTM